AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -703,6 +704,69 @@ - (instancetype)styledWithBlock:(void (^)(ASLayoutElementStyle *style))styleBloc
 
 #pragma mark - Layout
 
+- (void)invalidateSize
```

Comment:
It's an interesting case - as implemented today since it is automatically called, it's basically like "I know I need to be a different size -- deal with it"  rather than "I need my size recalculated"

I think setNeedsLayoutFromAbove is probably the best option for now.  I agree though that this is not important due to being internal; it's up to you if you want to adopt that name before landing or revisit later when we have better ideas.